Register-Share-System/src/Server/server_app.java

```diff
@@ -1,21 +1,67 @@
 package Server;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.InetAddress;
+
 public class server_app {
 
-	public static int localPort = 6066;		//server port
-	public static int destPort = 6055;		//client port
+	public static int localPort1 = 6077;	//hard coded port that first server listens on
+	public static int localPort2 = 6066;	//hard coded port that second server listens on
 	public static UDPServer server;
 	
 	public static void main(String[] args) {
 		
-		//start server
-		server = new UDPServer(localPort,destPort);
-		server.start();
+		//initialize console input
+		BufferedReader cin = new BufferedReader(new InputStreamReader(System.in));   
+		
+		//prompt user to designate server as first or second
+		System.out.println("please designate this server as server 1 or server 2");
+		System.out.print("enter 1 or 2 > ");
+		
+		try {
+			
+			int serverNum = Integer.parseInt(cin.readLine());
+			if(serverNum==1) {
+				
+				//start server, which will await sync with server 2 before coming online
+				server = new UDPServer(localPort1);
+				server.start();
+			
+			} else if(serverNum==2) {
+				
+				//solicit ip for server 1
+				System.out.println("please enter the second servers IP address, or 'localhost' for local machine");
```

Comment:
by second server you mean "the other server" right? like server 1? wording tripped me up